docs/docs/ab_testing.md

```diff
@@ -0,0 +1,95 @@
+Pathfora supports A/B testing on a global level as well as on a targeted audience. You can select to split a pool of users to show some proportion of them a group of modules "A" and the other portion a group "B", (group being a collection of 0 or more modules).
+
+
+``` javascript
+var moduleTest = pathfora.ABTest({
+  // A/B test settings
+});
+
+pathfora.initializeABTesting([ moduleTest ]);
+```
+
+<table>
+  <tr>
+    <td colspan="3" align="center"><code>ABTest</code> settings object</td>
+  </tr>
+  <tr>
+    <th>VALUE</th>
```

Comment:
This is true for other tables like this. The `VALUE` column is actually the `KEY` or `PROPERTY` column.